<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>combineList</class>
 <widget class="QDialog" name="combineList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>20</y>
     <width>731</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>配方号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Frst</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Second</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Third</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Forth</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Finished</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Occupation</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_Change">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change_SQL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
